src/styles/header.scss

```diff
@@ -0,0 +1,52 @@
+@import './variables';
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+}
+
+.nav {
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+    gap: 20px;
+  }
+
+  &__link {
+    display: block;
+    text-transform: uppercase;
+    font-weight: 500;
+    font-size: 12px;
+    text-decoration: none;
+    padding: 23px 0;
+    color: $color-main;
```

Comment:
Ensure that the variable `$color-main` is defined in the imported `variables` file. If it's not defined, this will cause a compilation error.